lib/webauth.js

```diff
@@ -207,6 +207,33 @@ module.exports = function(authManager, cookiePort, isSecurePort) {
     return;
   });
 
+  const _passwordResetInner = Promise.coroutine(function*(req, res, type) {
+   const handlers = getRelevantHandlers(authManager, req.body);
+   const result = new LoginResult();
+   for (const handler of handlers) {
```

Comment:
If you do this, it means that you are requesting a password reset from every handler.
The way this handler stuff is supposed to work is that maybe you have a handler for zos, facebook, and windows logins. You may want to login to all 3 at once. You do not want to password reset all 3 at once.
I think you should require that req.body defines exactly 1 handler, or that the URL contains a handler ID, or something like that.
Otherwise the user experience will be very strange.

Practically speaking, what will happen on 1.9 is you have 2 handlers: zss-auth and apiml-auth
zss-auth will do what you want.
apiml-auth will have canResetPassword == false, which will return success:false, which will make the UI show an error.
So, password reset will not work in that default setup. You need to limit it to zss for now.